gslib/commands/ls.py

```diff
@@ -337,10 +337,12 @@ def _PrintBucketInfo(self, bucket_blr, listing_style):
     # We're guaranteed by the caller that the root object is populated.
     bucket = bucket_blr.root_object
     location_constraint = bucket.location
+    location_type = bucket.locationType
     storage_class = bucket.storageClass
     fields = {
         'bucket': bucket_blr.url_string,
         'storage_class': storage_class,
+        'location_type': location_type,
```

Comment:
Can you add some logic to the long-listing tests in `gslib/tests/test_ls.py` to make sure this field shows up as expected?

Reply:
I may just need to wait on this one as the [API has the field available](https://www.googleapis.com/discovery/v1/apis/storage/v1/rest) but it isn't populating yet, apparently, so the  I'll circle back once this field starts populating.

Reply:
Sounds good.